app/webpacker/src/stylesheets/_customs_duties_explainer.scss

```diff
@@ -0,0 +1,13 @@
+.nhs-grey-5 {
```

Comment:
NHS?

Reply:
:D yeah, I've copied the CSS in the prototype, I'll see if I can reuse an existing color